Marlin/src/HAL/STM32F1/tft/tft_fsmc.h

```diff
@@ -30,10 +30,16 @@
 
 #include <libmaple/dma.h>
 
-#undef FSMC_DMA_DEV
-#undef FSMC_DMA_CHANNEL
-#define FSMC_DMA_DEV                      DMA1
-#define FSMC_DMA_CHANNEL               DMA_CH1
+#ifndef FSMC_DMA_DEV
+  #define FSMC_DMA_DEV      DMA2
+#else
+  #error "You are trying to use custom FSMC_DMA_DEV value. Remove this line to continue at your own risk."
```

Comment:
the errors are not mandatory/needed here...

Reply:
Do you still think it should be freely configurable, or you are ok with forcing DMA2_CH5 for all boards?

Reply:
yes, as i said, there are some free channels regarding the features used by the board... and it may help for the F4 maple board... else instead of #error you can use #warning

Reply:
Ok, though I sincerely hope that there will be no F4 maple board.

Reply:
yea, im not sure, STM32 framework now works well even on the F1, if the maintainer took the time to make the framework board...